var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -24,12 +30,14 @@ class Hdf5VolAsync(CMakePackage):
 
     depends_on("mpi")
     depends_on("argobots@main")
```

Comment:
@lrknox can this be made to `argobots@1.1:`. This is causing me headache somewhere else.

Reply:
Great, thanks!  It's in.